### PR TITLE
Update device-control-removable-storage-access-control.md

### PR DESCRIPTION
**Pull request by tewchen**

### CODE REVIEW

File: microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md [hunk 1]
```diff
@@ -90,8 +90,8 @@ You can use the following properties to create the access control policy:
...
-| **ComputerSID** | Local computer SID or computer SID group or the SID of the AD object, defines whether to apply this policy over a specific machine or machine group. One entry can have a maximum of one ComputerSID and an entry without any ComputerSID means applying the policy over the machine. If you want to apply an Entry to a specific user and specific machine, add both SID and ComputerSID into the same Entry. |  |
+| **Sid** | Local user Sid or user Sid group or the Sid of the AD object, defines whether to apply this policy over a specific user or user group. One entry can have a maximum of one SID and an entry without any SID means applying the policy over the machine. |  |
+| **ComputerSid** | Local computer Sid or computer Sid group or the Sid of the AD object, defines whether to apply this policy over a specific machine or machine group. One entry can have a maximum of one ComputerSid and an entry without any ComputerSID means applying the policy over the machine. If you want to apply an Entry to a specific user and specific machine, add both Sid and ComputerSid into the same Entry. |  |
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md:94]:
```suggestion
| **Sid** | Local user Sid or user Sid group or the Sid of the AD object, defines whether to apply this policy over a specific user or user group. One entry can have a maximum of one Sid and an entry without any Sid means applying the policy over the machine. |  |
| **ComputerSid** | Local computer Sid or computer Sid group or the Sid of the AD object, defines whether to apply this policy over a specific machine or machine group. One entry can have a maximum of one ComputerSid and an entry without any ComputerSid means applying the policy over the machine. If you want to apply an Entry to a specific user and specific machine, add both Sid and ComputerSid into the same Entry. |  |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @tewchen, kindly commit the suggested change. Many thanks.

**Comment by yogkumgit** [comment]
@tewchen Please implement the suggested changes. Thanks

**Comment by yogkumgit** [comment]
@tewchen Gentle reminder

**Comment by yogkumgit** [comment]
@tewchen closing this PR, as not updated as advised. Please feel free to reopen it, if you wish to take an action. Thanks

**Pull request closed without merging**